src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1470,5 +1469,5 @@
        (str
         "Unable to install optimal indexing\n\n"
         "We are unable to create optimal indexes for your database.\n"
-        "For maximum index performance, we recommend using PostgreSQL 9.3 or\n"
+        "For maximum index performance, we recommend using PostgreSQL 9.4 or\n"
```

Comment:
I'm not sure the version is needed here, since the check for version is no longer valid. In fact this error message now becomes specific to HSQLDB, so we can call that out now "We are unable to create optimal indexes for HSQLDB ..." etc.